assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -223,15 +224,18 @@ function Title({
       )}
       <Toggle
         toggle={toggle}
-        disabled={!canCollapse}
+        disabled={!canCollapse || isDisabledToggle}
+        isDisabledToggle={isDisabledToggle}
```

Comment:
this is confusing to me. Are there two different disabled states?

If one represents how it must be disabled and `collapsed` then I'd say we should just have two different pieces of state:
- `disabled` which decides whether the toggle can be interacted with
- `collapsed` which would mean whether the toggle is open or not

I could see `collapsed` overriding the `disabled` state, but I think having both of these is a little confusing.

Reply:
hmmm, so this is meant more as 2 ways to say 'hey this toggle shouldn't toggle' and to just leave it be in whatever state it currently exists in. Up until now we've never had disabled toggles that are collapsed. If `canCollapse` is true but the toggle itself isn't meant to be toggled yet I want the `isDisabledToggle` to override that `canCollapse` value. Similarly, if `canCollapse` is false (making the disabled prop true) then I don't care what `isDisabledToggle` is because it no longer is relevant.

Reply:
Gotcha. Still, it feels like having two disabled states: `disabled` and `isDisabledToggle` is redundant for the `SimplePanel` 

I'd say we rename the `isDisabledToggle` to be something that doesn't use the word `disabled` or we combine the states to just go in as one prop

Reply:
Fair argument, I updated it to make it clearer. I still think there's a lot going on with it so I don't want to combine any further, but I did add some notes and new variables so that it makes more sense hopefully. Lemme know what ya think.